inference-engine/src/transformations/include/transformations/smart_reshape/utils.hpp

```diff
@@ -7,14 +7,15 @@
 #include <transformations_visibility.hpp>
 #include <ngraph/op/util/op_annotations.hpp>
 #include <ngraph/opsets/opset4.hpp>
```

Comment:
If opset7::ShapeOf is equal to opset4::ShapeOf then remove opset4 include.

Reply:
Returned changes. Not actual anymore after GatherBase implemented